packages/jaeger-ui/src/model/ddg/transformTracesToPaths.test.js

```diff
@@ -45,109 +61,111 @@ describe('transform traces to ddg paths', () => {
       traceID,
```

Comment:
I'm a bit confused by how service names are assigned in L55. Looks like a string `"{first word of span id} service"`, but the first word of span is, per L47, is span name. So it seems _every_ span in these tests will have a different service name.

Reply:
Yes every span gets a unique service and operation name. currently service name is not used as part of the trace to path calculation. once we settle on a specification for the flink job we will apply the same rule here and likely need to tweak the test.